src/main/java/com/oltpbenchmark/benchmarks/wikipedia/data/RevisionHistograms.java

```diff
@@ -290,12 +290,16 @@ public abstract class RevisionHistograms {
      */
     public static final int[] REVISION_DELTA_SIZES = {1000, 10000, 100000};
 
+    /**
+     * Simple generic class overload to avoid some cast warnings below.
```

Comment:
Should this type of handling be part of a best practices doc? I.e., avoid casting, use some structures like this instead.

Reply:
Not a bad idea.  It's basically a limitation in the java language spec as I understand it.  You can't create arrays of a generics:
https://docs.oracle.com/javase/tutorial/java/generics/restrictions.html#createArrays